src/components/interfaces.ts

```diff
@@ -2,8 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 export interface ComponentDefinition {
   name: string;
+  /** @deprecated */
   releaseStatus: string;
+  /** @deprecated */
   version?: string;
+  /** @deprecated */
   description?: string;
```

Comment:
We have no use of these features on our documentation website, so it is better to mark them as deprecated and eventually remove